### ISSUE TITLE
Question autour de nouveau rôle auteur pour les métadonnées credit, tuteur et referent

### ISSUE THREAD

**Issue by Corentinfaye** [CCSDForge/hal-tei-schema#3]
Les métadonnées credit, tuteur et référent ne sont actuellement pas comme auteur car ils ne sont pas géré comme tel. 

La bonne solution serait de les avoir sous la forme :

TEI/text/body/listBibl/biblFull/titleStmt/author[role=@”credit”]

Mais il y a la question de créer des auteurs qui ne sont pas dans le référentiel auteur de la base. Est-ce qu'on accepte de faire cela malgré l'absence des informations dans le référentiels auteurs où on le place ailleurs.

**Comment by marmol** [comment]
En fait, ce n'et pas le probleme de creation que je relevait, ce sont les valeur actuelle de ce champs:
     Musée de la sidérurgie en Bourgogne du Nord
     Jean-Paul Deler, CEGET/CNRS
     Ernest Will, Robert Amy ?

On ne sait pas structurer l'information dans les champs normaux de <author>
ON fait quoi: 
 juste <author role="credit">Ma chaine</author>    ?

**Comment by laurentromary** [comment]
Oui, pour l'instant on ne change rien. Je verrai bien anticiper une transition à discuter où on offre au niveau de l'interface la possibilité d'entrée une vraie personne ou une vraie structure. Mais, pour l'heure. On unifie juste le format d'export/import (sans tout casser car on a l'attribut role)
